Zachet/Zachet/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace Zachet
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите натуральное число");
+            int n = new int();
+            while (true)
+            { 
+                var UserNumb = (Console.ReadLine());

```

Comment:
Стиль! Локальные переменные принято называть с маленькой буквы.
```suggestion
                var userNumb = (Console.ReadLine());

```